### ISSUE TITLE
Wrong colors.

### ISSUE THREAD

**Issue by FranciscoJavierDiazBlanco** [MiSTer-devel/ZX-Spectrum_MISTer#34]
The dark tones of the colors are too dark.   
Core zx spectrum photo:
 https://ibb.co/TR2wFvN
Core ZXNext Photo: 
https://ibb.co/WDyBPth

**Comment by jonpatterson** [comment]
I can confirm I have the same issue.  I've attached two images of the colour test screen, one captured from the ZX Spectrum core and the other from the ZX Spectrum Next core.  From what I can see it looks like the _bright_ half of the palette is pretty similar, but the _basic_ half is too dark.  The ZX Spectrum Next core seems to accurately represent the colours as I'd expect.

**ZX Spectrum**
![zx_spectrum](https://user-images.githubusercontent.com/26647345/150967516-117fc2f9-9d1b-436f-9f5f-4c8f447e177d.png)
**ZX Spectrum Next**
![zx_spectrum_next](https://user-images.githubusercontent.com/26647345/150967529-ab73cf95-8627-49dc-b4f9-e6a6a720a97d.png)

**Comment by sorgelig** [comment]
try this build:
[ZX-Spectrum.zip](https://github.com/MiSTer-devel/ZX-Spectrum_MISTer/files/7944310/ZX-Spectrum.zip)

**Comment by jonpatterson** [comment]
> try this build: [ZX-Spectrum.zip](https://github.com/MiSTer-devel/ZX-Spectrum_MISTer/files/7944310/ZX-Spectrum.zip)

A massive improvement!  The same colour test screen on the ZX Spectrum and ZX Spectrum Next cores are very similar now.

I took two screenshots from the cores again and compared the hex values.  For the basic colours they are identical, with the exception of magenta which is slightly different.  Comparing the bright values, if anything the ZX Spectrum core looks a little brighter than the ZX Next core now!  Interestingly I also took a screenshot of the same test screen in the Fuse emulator and it's identical to the ZX Spectrum build you provided.

Many thanks Alexey, this looks good to me 👍 

**ZX Spectrum Next**
![zx_next_test](https://user-images.githubusercontent.com/26647345/151242008-e5043441-fb82-4945-af38-2dde697f9d7c.png)

**ZX Spectrum**
![zx_spectrum_test](https://user-images.githubusercontent.com/26647345/151242021-6dc385d8-140e-4ecb-bde1-4b89372c9ae5.png)

**Fuse Emulator**
![fuse_test](https://user-images.githubusercontent.com/26647345/151243481-bfb427e9-cdce-4a5a-bd76-d444cd96fdfb.png)
